scripts/ci-e2e.sh

```diff
@@ -86,22 +87,21 @@ create_powervs_network_instance(){
     ibmcloud pi service-target ${CRN}
 
     # Create the network instance
-    ibmcloud pi network-create-public ${IBMPOWERVS_NETWORK_NAME} --dns-servers "8.8.8.8 9.9.9.9"
+    capibmadm powervs network create --name ${IBMPOWERVS_NETWORK_NAME} --service-instance-id ${IBMPOWERVS_SERVICE_INSTANCE_ID} --zone ${ZONE}
 
 }
 
 init_network_powervs(){
-    install_pvsadm
+    install_capibmadm
     create_powervs_network_instance
 
     # Creating ports using the pvsadm tool
-    ./pvsadm create port --description "capi-port-e2e" --network ${IBMPOWERVS_NETWORK_NAME} --instance-id ${IBMPOWERVS_SERVICE_INSTANCE_ID}
+    capibmadm powervs port create --network ${IBMPOWERVS_NETWORK_NAME} --description "capi-e2e" --service-instance-id ${IBMPOWERVS_SERVICE_INSTANCE_ID} --zone ${ZONE}
 
     # Get and assign the IPs to the required variables
-    NEW_PORT=$(./pvsadm get ports --network ${IBMPOWERVS_NETWORK_NAME} --instance-id ${IBMPOWERVS_SERVICE_INSTANCE_ID} | sed -n '4 p')
-    PORT_ID="$(echo ${NEW_PORT} | cut -d'|' -f6 | xargs )"
-    export IBMPOWERVS_VIP="$(echo ${NEW_PORT} | cut -d'|' -f4 | xargs )"
-    export IBMPOWERVS_VIP_EXTERNAL="$(echo ${NEW_PORT} | cut -d'|' -f3 | xargs )"
+    NEW_PORT=$(capibmadm powervs port list --service-instance-id ${IBMPOWERVS_SERVICE_INSTANCE_ID} --zone ${ZONE} --network ${IBMPOWERVS_NETWORK_NAME} -o json)
+    export IBMPOWERVS_VIP="$(echo ${NEW_PORT} | jq -r '.items[0].ipAddress')"
+    export IBMPOWERVS_VIP_EXTERNAL="$(echo ${NEW_PORT} | jq -r '.items[0].externalIP')"
```

Comment:
hope you don't run with index out of range issue here!

Reply:
Ideally, if the port creation succeeds, this would work fine. Added a check to be sure it would work as expected. PTAL.